### PR TITLE
Deployment to Optimism Sepolia

### PR DESCRIPTION
**Pull request by kovalgek**
Deploy bridge executor for Optimism Sepolia testnet

### CODE REVIEW

**Comment by TheDZhon** [comment]
Seems there were no secrets for Arbitrum and Optimism to verify the bytecode.
Fixed this :+1:

**Review by arwer13 — APPROVED** [review]
LGTM, thanks!

Could refactor a bit in some place to replace specific testnet name by just "testnet", but let's opt out of this for now.

File: helpers/gov-constants.ts [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-  OVM_L1_MESSENGER_MAIN: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1',
-  OVM_L2_MESSENGER: '0x4200000000000000000000000000000000000007',
   // The deployment process includes testing the deployed governance bridge via calling the
   // 'updateEthereumGovernanceExecutor' method. As the initial value for EthereumGovExecutor
   // is used test instance of Aragon Agent on mainnet. It will be replaced with the official
   // Lido Aragon Agent before usage.
-  ETHEREUM_GOV_EXECUTOR: '0x184d39300f2fa4419d04998e9c58cb5de586d879',
+
+  // Common
+  ETHEREUM_GOV_EXECUTOR: '0x32A0E5828B62AAb932362a4816ae03b860b65e83',
```
Comment (TheDZhon) [inline helpers/gov-constants.ts:8]: Why do we change this addr from mainnet to sepolia?

File: helpers/gov-constants.ts [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
   OVM_GUARDIAN: '0x0000000000000000000000000000000000000000',
+  OVM_L2_MESSENGER: '0x4200000000000000000000000000000000000007',
+
+  OVM_L1_MESSENGER_MAIN: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1',
+
+  // Arbitrum
   ARB_GUARDIAN: '0x0000000000000000000000000000000000000000',
   RETRYABLE_TICKET_TX_ADDRESS: '0x000000000000000000000000000000000000006E',
+  INBOX_MAIN: '0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f',
+  INBOX_RINKEBY: '0x578BAde599406A8fE3d24Fd7f7211c0911F5B29e',
```
Comment (TheDZhon) [inline helpers/gov-constants.ts:22]: I am not sure that RINKEBY is still alive

File: helpers/gov-constants.ts [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+  // Common
+  ETHEREUM_GOV_EXECUTOR: '0x32A0E5828B62AAb932362a4816ae03b860b65e83',
+
+  // Optimism
+  // https://docs.optimism.io/chain/addresses
+  OVM_L1_MESSENGER_SEPOLIA: '0x58Cc85b8D04EA49cC6DBd3CbFFd00B4B8D6cb3ef',
   OVM_GUARDIAN: '0x0000000000000000000000000000000000000000',
+  OVM_L2_MESSENGER: '0x4200000000000000000000000000000000000007',
+
+  OVM_L1_MESSENGER_MAIN: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1',
```
Comment (TheDZhon) [inline helpers/gov-constants.ts:16]: maybe 'MAINNET'?

File: helpers/gov-constants.ts
Comment (TheDZhon) [inline helpers/gov-constants.ts:1]: btw, should be remove not only Kovan, but Rinkeby with this PR?

**Review by TheDZhon — CHANGES_REQUESTED** [review]
:+1: 

Thank you for keeping the things up-to-date, I'd suggest a bit more polishing :pray:

File: hardhat.config.ts [hunk 4]
```diff
@@ -111,7 +111,26 @@ const hardhatConfig: HardhatUserConfig = {
...
+        },
+        {
+            network: 'optimism-testnet',
+            chainId: 11155420,
+            urls: {
+              apiURL: 'https://api-sepolia-optimism.etherscan.io/api',
+              browserURL: 'https://sepolia-optimism.etherscan.io',
+            },
+          },
+      ],
```
Comment (TheDZhon) [inline hardhat.config.ts:133]: Please consider formatting this block with a more accurate indentation :eyes:

File: hardhat.config.ts [hunk 5]
```diff
@@ -111,7 +111,26 @@ const hardhatConfig: HardhatUserConfig = {
     apiKey: {
       optimisticEthereum: OPTIMISTIC_ETHERSCAN_KEY,
       arbitrumOne: ARBISCAN_KEY,
+      "optimism-testnet": OPTIMISTIC_ETHERSCAN_KEY,
```
Comment (TheDZhon) [inline hardhat.config.ts:114]:
```suggestion
      "optimisticSepolia": OPTIMISTIC_ETHERSCAN_KEY,
```

File: hardhat.config.ts [hunk 6]
```diff
@@ -111,7 +111,26 @@ const hardhatConfig: HardhatUserConfig = {
...
+        {
+          network: 'sepolia',
+          chainId: 11155111,
+          urls: {
+            apiURL: 'https://api-sepolia.etherscan.io/api',
+            browserURL: 'https://sepolia.etherscan.io',
+          },
+        },
+        {
+            network: 'optimism-testnet',
```
Comment (TheDZhon) [inline hardhat.config.ts:126]:
```suggestion
            network: 'optimism-sepolia',
```

File: helpers/gov-constants.ts [hunk 7]
```diff
@@ -1,17 +1,25 @@
...
-  OVM_L1_MESSENGER_KOVAN: '0x4361d0F75A0186C05f971c566dC6bEa5957483fD',
-  OVM_L1_MESSENGER_MAIN: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1',
-  OVM_L2_MESSENGER: '0x4200000000000000000000000000000000000007',
   // The deployment process includes testing the deployed governance bridge via calling the
   // 'updateEthereumGovernanceExecutor' method. As the initial value for EthereumGovExecutor
   // is used test instance of Aragon Agent on mainnet. It will be replaced with the official
   // Lido Aragon Agent before usage.
+
+  // Common
   ETHEREUM_GOV_EXECUTOR: '0x184d39300f2fa4419d04998e9c58cb5de586d879',
```
Comment (TheDZhon) [inline helpers/gov-constants.ts:8]:
```suggestion
  ETHEREUM_GOV_EXECUTOR_MAIN: '0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c',
  ETHEREUM_GOV_EXECUTOR_SEPOLIA: '0x32A0E5828B62AAb932362a4816ae03b860b65e83',  
```

File: helpers/gov-constants.ts [hunk 8]
```diff
@@ -1,17 +1,25 @@
...
   OVM_GUARDIAN: '0x0000000000000000000000000000000000000000',
+  OVM_L2_MESSENGER: '0x4200000000000000000000000000000000000007',
+
+  OVM_L1_MESSENGER_MAIN: '0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1',
+
+  // Arbitrum
   ARB_GUARDIAN: '0x0000000000000000000000000000000000000000',
   RETRYABLE_TICKET_TX_ADDRESS: '0x000000000000000000000000000000000000006E',
+  INBOX_MAIN: '0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f',
+  INBOX_RINKEBY: '0x578BAde599406A8fE3d24Fd7f7211c0911F5B29e',
```
Comment (TheDZhon) [inline helpers/gov-constants.ts:22]: why do we still have `RINKEBY` tho?

Comment (TheDZhon) [reply helpers/gov-constants.ts:22]: should it be `INBOX_SEPOLIA`?

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Thank you for improvements :pray: 

Yet i'd suggest giving it one more shot :gun:

File: hardhat.config.ts (hunk 6 above)
Comment (kovalgek) [reply hardhat.config.ts:126]: I made it "optimisticSepolia" to be in sync with apiKey

**Review by arwer13 — APPROVED** [review]

**Pull request merged**